modules/Hoot/ui/sidebar/layerConflate.js

```diff
@@ -18,10 +18,13 @@ class LayerConflate extends SidebarForm {
     }
 
     async getData() {
-        let that = this;
-        await Hoot.api
-            .getConflateTypes()
-            .then(conflateTypes => that.conflateTypes = conflateTypes);
+        this.conflateTypes = await Hoot.api.getConflateTypes();
+        const favOpts = await Hoot.api.getFavoriteAdvOpts();
+        if ( favOpts ) {
```

Comment:
is this a safe enough check? like will return null or an empty list from the api request?